### PR TITLE
fix(notification): align with https://github.com/frappe/frappe/pull/22595

### PR DESCRIPTION
**Pull request by blaggacao**
Companion to https://github.com/frappe/frappe/pull/22595

### CODE REVIEW

**Comment by blaggacao** [comment]
Upstream was merged already. :smile:

**Comment by blaggacao** [comment]
This is ready to merge (before force-staled, please).

File: erpnext/accounts/notification/notification_for_new_fiscal_year/notification_for_new_fiscal_year.json [hunk 1]
```diff
@@ -11,19 +11,21 @@
...
  "name": "Notification for new fiscal year",
  "owner": "Administrator",
  "recipients": [
   {
-   "email_by_role": "Accounts User"
+   "receiver_by_document_field": "Accounts User"
   },
   {
-   "email_by_role": "Accounts Manager"
+   "receiver_by_document_field": "Accounts Manager"
```
Comment (barredterra) [inline erpnext/accounts/notification/notification_for_new_fiscal_year/notification_for_new_fiscal_year.json]: These don't seem like document fields to me 🤔

File: erpnext/accounts/notification/notification_for_new_fiscal_year/notification_for_new_fiscal_year.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<h3>{{_("Fiscal Year")}}</h3>
```
Comment (barredterra) [inline erpnext/accounts/notification/notification_for_new_fiscal_year/notification_for_new_fiscal_year.html]:
```suggestion
<h3>{{ _("Fiscal Year") }}</h3>
```

File: erpnext/manufacturing/notification/material_request_receipt_notification/material_request_receipt_notification.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<p><b>Material Request Type</b>: {{ doc.material_request_type }}<br>
+<b>Company</b>: {{ doc.company }}</p>
+
+<h3>Order Summary</h3>
+
+<table border=2 >
+    <tr align="center">
+        <th>Item Name</th>
+        <th>Received Quantity</th>
+    </tr>
```
Comment (barredterra) [inline erpnext/manufacturing/notification/material_request_receipt_notification/material_request_receipt_notification.html]:
```suggestion
<p><b>{{ _("Material Request Type") }}</b>: {{ doc.material_request_type }}<br>
<b>{{ _("Company") }}</b>: {{ doc.company }}</p>

<h3>{{ _("Order Summary") }}</h3>

<table border=2 >
    <tr align="center">
        <th>{{ _("Item Name") }}</th>
        <th>{{ _("Received Quantity") }}</th>
    </tr>
```

**Review by barredterra — CHANGES_REQUESTED** [review]

File: erpnext/accounts/notification/notification_for_new_fiscal_year/notification_for_new_fiscal_year.json (hunk 1 above)
Comment (blaggacao) [reply erpnext/accounts/notification/notification_for_new_fiscal_year/notification_for_new_fiscal_year.json]: It's only in js:
```console
frappe/email/doctype/notification/notification.js
72-				});
73-			}
74-
75-			// set email recipient options
76-			frm.fields_dict.recipients.grid.update_docfield_property(
77:				"receiver_by_document_field",
78-				"options",
79-				[""].concat(["owner"]).concat(receiver_fields)
80-			);
81-		});
82-	},
```

**Comment by blaggacao** [comment]
@barredterra Nice! Thank you for the translations! :+1:

**Comment by blaggacao** [comment]
I think we'd be ready to merge on this?

**Comment by blaggacao** [comment]
unstale

**Comment by deepeshgarg007** [comment]
@Mergifyio update

**Comment by blaggacao** [comment]
@Mergifyio update

File: erpnext/accounts/notification/notification_for_new_fiscal_year/notification_for_new_fiscal_year.json (hunk 1 above)
Comment (barredterra) [reply erpnext/accounts/notification/notification_for_new_fiscal_year/notification_for_new_fiscal_year.json]: I didn't get what you're saying. My point was that `receiver_by_document_field` should most likely contain a valid field name, not a role. For roles we have `receiver_by_role`.

Comment (blaggacao) [reply erpnext/accounts/notification/notification_for_new_fiscal_year/notification_for_new_fiscal_year.json]: Ah damn! Misunderstanding slumbering here. :smile: You are absolutely right. :+1:

Comment (blaggacao) [reply erpnext/accounts/notification/notification_for_new_fiscal_year/notification_for_new_fiscal_year.json]: Now we should be good: https://github.com/frappe/erpnext/pull/38157/commits/d86186ec472bac0556c5815d28259a2b0fa6c90d &mdash; but the automerge was disabled.

**Comment by blaggacao** [comment]
@deepeshgarg007: @barredterra and myself had another round of review. It should be finally good now.

**Review by barredterra — APPROVED** [review]

**Pull request merged**